projects/core/src/user/facade/consent.service.ts

```diff
@@ -0,0 +1,72 @@
+import { Injectable } from '@angular/core';
```

Comment:
shouldn't this file be in the anonymousconsent folder in storefrontlib and not the facade since it acts like a helper function

Reply:
When is this file and functions used?

Reply:
I'd say this is also a facade, that combines two other facades - anonymous and user-consent facades.
It's being prepared for Corey's profiletag, and it's going to be used in the CDS lib.